pysoma/dataframe.py

```diff
@@ -6,7 +6,9 @@
 from collections import namedtuple
 
 import pyspark.sql.functions as F
-from pyspark.sql import SparkSession, DataFrame, Column
+from pyspark.sql import SparkSession, DataFrame
+from pyspark.sql.column import Column
```

Comment:
Any particular reason to move the `Column` import? No big deal either way

Reply:
I had a brief implementation using PySpark `_func`s from the column module - I had lots of additional imports. Then I realized the DataFrame API now contains `filter` and none of that was required, removed them, just didn't move Column back - no hard opinion here either just YOLO'd it and left it on that line.